src/test/java/net/openhft/hashing/LongHashFunctionTest.java

```diff
@@ -56,36 +58,48 @@ public static void testBoolean(LongHashFunction f, int len) {
         for (boolean b : new boolean[] {true, false}) {
             boolean[] a = {b};
             long single = f.hashBoolean(b);
-            assertEquals(single, f.hashBooleans(a));
+            long array = f.hashBooleans(a);
+            assertEquals(single, array);
             assertEquals(single, f.hash(a, UnsafeAccess.unsafe(), UnsafeAccess.BOOLEAN_BASE, 1L));
         }
     }
 
     private static void testPrimitives(LongHashFunction f, long eh, int len, ByteBuffer bb) {
-        if (len == 1)
-            assertEquals("byte hash", eh, f.hashByte(bb.get(0)));
-
+        long actual;
+        if (len == 1) {
+            actual = f.hashByte(bb.get(0));
+            assertEquals("byte hash", eh, actual);
+        }
 
         if (len == 2) {
-            assertEquals("short hash", eh, f.hashShort(bb.getShort(0)));
-            assertEquals("char hash", eh, f.hashChar(bb.getChar(0)));
+            actual = f.hashShort(bb.getShort(0));
+            assertEquals("short hash", eh, actual);
+            actual = f.hashChar(bb.getChar(0));
+            assertEquals("char hash", eh, actual);
         }
 
-        if (len == 4)
-            assertEquals("int hash", eh, f.hashInt(bb.getInt(0)));
-
-        if (len == 8)
-            assertEquals("long hash", eh, f.hashLong(bb.getLong(0)));
+        if (len == 4) {
+            actual = f.hashInt(bb.getInt(0));
+            assertEquals("int hash", eh, actual);
+        }
+        if (len == 8) {
+            actual = f.hashLong(bb.getLong(0));
+            assertEquals("long hash", eh, actual);
+        }
     }
 
     private static void testNegativePrimitives(LongHashFunction f) {
         byte[] bytes = new byte[8];
         Arrays.fill(bytes, (byte) -1);
-        assertEquals("byte hash", f.hashBytes(bytes, 0, 1), f.hashByte((byte) -1));
-        assertEquals("short hash", f.hashBytes(bytes, 0, 2), f.hashShort((short) -1));
-        assertEquals("char hash", f.hashBytes(bytes, 0, 2), f.hashChar((char) -1));
-        assertEquals("int hash", f.hashBytes(bytes, 0, 4), f.hashInt(-1));
-        assertEquals("long hash", f.hashBytes(bytes), f.hashLong(-1L));
+        long oneByteActual = f.hashBytes(bytes, 0, 1);
```

Comment:
Looks like "actual" params go first in this block, but last in the block above (lines 81-88)